LayoutTests/fast/forms/textfield-onchange-without-focus.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <script>
-function test() {
+onload = async ()=> {
+  testRunner?.waitUntilDone();
```

Comment:
The state of the art for reftests that are async is using the `reftest-wait` class on the element and then removing it when the test is done

Reply:
Fixed.